### PR TITLE
Add time travel arrows

### PR DESCRIPTION
**Pull request by krsjoseph**
[Notion Task](https://www.notion.so/lncapital/Arrows-to-quickly-jump-timeframes-33ccbaab3d8c404aab0c1335a6cfd59b)

https://user-images.githubusercontent.com/1019869/162409628-800a0c7a-5843-4702-bc9e-874730f40f97.mp4

### CODE REVIEW

File: web/src/components/timeIntervalSelect/interval_select.scss [hunk 1]
```diff
@@ -168,7 +168,7 @@
   }
 }
 
-.torq .time-interval-wrapper {
+.torq #time-interval-wrapper {
```
Comment (ohenrik) [inline web/src/components/timeIntervalSelect/interval_select.scss:171]: is `time-interval-wrapper` applied as both an id and a class? Or where the css bellow not applied before?

**Comment by ohenrik** [comment]
This looks good!

File: web/src/components/timeIntervalSelect/interval_select.scss (hunk 1 above)
Comment (krsjoseph) [reply web/src/components/timeIntervalSelect/interval_select.scss:171]: I changed this to an ID to have more specificity  and not use any `!important` 's

**Comment by krsjoseph** [comment]
Changes addressed @ohenrik (y)

**Pull request merged**